pkg/sql/compile/compile.go

```diff
@@ -4019,6 +4019,9 @@ func (c *Compile) generateNodes(n *plan.Node) (engine.Nodes, []any, []types.T, e
 		if err != nil {
 			return nil, nil, nil, err
 		}
+		if n.TableDef.Name == "defect_history" {
```

Comment:
All these n.TableDef.Name == "xxx" can be replaced by if session var debug_moc_4319 is true, ...